modules/backend/models/Preference.php

```diff
@@ -159,6 +159,7 @@ public function getLocaleOptions()
             'ru' => [Lang::get('system::lang.locale.ru'), 'flag-ru'],
             'sv' => [Lang::get('system::lang.locale.sv'), 'flag-se'],
             'tr' => [Lang::get('system::lang.locale.tr'), 'flag-tr'],
+            'uk' => [Lang::get('system::lang.locale.ua'), 'flag-ua'],
```

Comment:
You have a typo here. It should be `Lang::get('system::lang.locale.uk')`, not `lang.locale.ua`